packages/docs/src/utilities/utils.ts

```diff
@@ -1,25 +1,39 @@
 // File for documentation helper functions
 
+function getUIElementNameFromPath (path: string): string {
```

Comment:
Very nitpicky, but you can use `const getUIElementNameFromPath = () => {}` here 🤓